code/kris/html_css_flask/lab-02/templates/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="{{url_for('static', filename = 'styles.css')}}">
+
+
+    <title>Kris' Blog about Otto</title>
+</head>
+<body>
+    <header> Kris' Blog About Otto Things</header>
+    <div class="topnav">
```

Comment:
Could incorporate the use of semantic elements, for example here could be `nav` instead of `div`

https://www.w3schools.com/html/html5_semantic_elements.asp